src/FilterPanel.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+
+interface FilterPanelProps {
+  setFilterMode: React.Dispatch<React.SetStateAction<string>>,
+  filteredTodos: Todo[];
+}
+
+export const FilterPanel: React.FC<FilterPanelProps> = ({
+  setFilterMode, filteredTodos,
+}) => {
+  const [activeButton, setActiveButton] = useState<string>('All');
```

Comment:
Pls check [task checklist point 10](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md#task-checklist) and make a separate const for types)